lib/web_ui/dev/test_platform.dart

```diff
@@ -352,13 +352,7 @@ Golden file $filename did not match the image generated by the test.
       message.writeln('Golden file: ${expectedFile.path}');
       message.writeln('Actual file: ${actualFile.path}');
 
-      if (diff.rate < maxDiffRateFailure) {
-        // Issue a warning but do not fail the test.
-        print('WARNING:');
-        print(message);
-        return 'OK';
-      } else {
-        // Fail test
+      if (diff.rate > maxDiffRateFailure) {
```

Comment:
It probably doesn't matter much, but if you want to keep the exact same behavior, it should be `diff.rate >= maxDiffRateFailure`